Dockerfile

```diff
@@ -1,27 +1,36 @@
-FROM python:3.7-buster
+FROM python:3.7-slim-buster AS compile-image
+LABEL maintainer="Wazo Maintainers <dev@wazo.community>"
 
-ENV DEBIAN_FRONTEND noninteractive
+RUN python -m venv /opt/venv
+# Activate virtual env
+ENV PATH="/opt/venv/bin:$PATH"
 
-ADD . /usr/src/wazo-calld
-ADD ./contribs/docker/certs /usr/share/xivo-certs
+ENV DEBIAN_FRONTEND noninteractive
 
+COPY . /usr/src/wazo-calld
 WORKDIR /usr/src/wazo-calld
+RUN pip install -r requirements.txt
+RUN python setup.py install
+
+FROM python:3.7-slim-buster AS build-image
+COPY --from=compile-image /opt/venv /opt/venv
+
+COPY ./etc/wazo-calld /etc/wazo-calld
+COPY ./contribs/docker/certs /usr/share/xivo-certs
+COPY ./bin/wazo-pdf2fax /usr/bin/wazo-pdf2fax
 
 RUN true \
-    && apt-get -qq update \
-    && apt-get -qq -y install libpq-dev libyaml-dev ghostscript \
-    && pip install -r requirements.txt \
-    && python setup.py install \
-    && cp -av etc/wazo-calld /etc \
-    && install -m 755 -o root -g www-data bin/wazo-pdf2fax /usr/bin/wazo-pdf2fax \
+    && apt-get -q update \
```

Comment:
Is the `DEBIAN_FRONTEND` environment variable necessary in the first image or this one? It is defined in the first image but `apt-get` is used in the second image.

Reply:
nice catch, well since it doesn't seem to change anything, I removed it